<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetNota</class>
 <widget class="QWidget" name="WidgetNota">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>112</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitulo">
     <property name="text">
      <string>Nota</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spbNota"/>
   </item>
   <item>
    <widget class="QCheckBox" name="chkActivo">
     <property name="text">
      <string>Activo</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
